playbook/app/pb_kits/playbook/pb_docs/kit_example.rb

```diff
@@ -33,7 +33,17 @@ def highlighter
       def source
         @source ||= begin
           extension = type == "react" ? "jsx" : "html.erb"
-          read_kit_file("_#{example_key}.#{extension}")
+          raw_code = read_kit_file("_#{example_key}.#{extension}")
+          raw_code = raw_code.gsub("../../", "playbook-ui")
+          raw_code = dark ? raw_code.gsub("{...props}", "dark") : raw_code.gsub(/\s*{...props}\s*\n/, "\n")
+          puts raw_code == /props:\s*{.*}\s*\)/m
+          puts Time.current
+          if raw_code.include?("props: { ")
```

Comment:
because lines shouldn't end with empty characters, if the string includes "props: { " it means the props are all in the same line. So i don't add "\n" line breaks because that would be too hard to implement